src/containers/Login.js

```diff
@@ -107,6 +107,9 @@ export default class Login extends React.Component {
 
     if (error === 'no user') error = 'Login was canceled or no user data was found.'
     if (error === 'no email') error = 'The user data did not include an email address.'
+    if (error && error.match(/password account not found. available: \[(.*)\]/)) {
+      error = <span>Your account has no password set. <a href='/set-password'>Set your password</a></span>
+    }
```

Comment:
can you parse the "available" section and display messages like "Or use Facebook" as appropriate, so this doesn't confuse people who use third-party auth?